packages/mobile/App.svelte

```diff
@@ -25,12 +26,36 @@
     import { initialiseRouterManager, RouterManagerExtensionName } from '@core/router'
 
     import { DashboardView, LoginRouter, OnboardingRouter } from '@views'
-    import { closeAllDrawers } from '@/auxiliary/drawer'
+    import { closeAllDrawers, drawers } from '@/auxiliary/drawer'
 
     appStage.set(AppStage[process.env.STAGE.toUpperCase()] ?? AppStage.ALPHA)
 
     checkAndMigrateProfiles()
 
+    /**
+     * Handle Android top status bar (not needed for iOS)
+     * @todo remove when implement status bar overlay
+     * https://github.com/iotaledger/firefly/issues/6345
+     */
+    $: if ($drawers[0]?.id !== 'profile') {
+        if ($appSettings.darkMode) {
+            void StatusBar.setBackgroundColor({ color: '#1B2D4B' })
```

Comment:
to avoid hardcoding the colors, you could try importing tailwind

```
import tailwindConfig from 'shared/tailwind.config.js'
import resolveConfig from 'tailwindcss/resolveConfig'

const configColors = resolveConfig(tailwindConfig).theme.colors
```
so you can do here:
```suggestion
            void StatusBar.setBackgroundColor({ color: configColors['gray']['50'] })
```

Reply:
and apply the same to the other colors, wdyt?

Reply:
I see this thread was resolved but the changes were not implemented, wdyt about these changes and apply them to all the colors harcoded here?

Reply:
I tried but couldn't get it to work on iOS. Since it's just a workaround and will be removed when https://github.com/iotaledger/firefly/issues/6345 is complete, I think we could merge as is